entrypoints/webcomponents-sd-ce-pf-index.js

```diff
@@ -23,4 +23,5 @@ import '../node_modules/@webcomponents/shadydom/src/shadydom.js';
 import '../node_modules/@webcomponents/custom-elements/src/custom-elements.js';
 import '../node_modules/@webcomponents/shadycss/entrypoints/scoping-shim.js';
 import '../node_modules/@webcomponents/url/url.js';
+import '../src/baseuri.js';
```

Comment:
Is this not meant to be imported in other entrypoints?

Reply:
No, this is only needed for IE 11